src/lib/Gen/adoc/AdocGenerator.cpp

```diff
@@ -75,7 +75,7 @@ build(
 
     auto errors = ex->wait();
     if(! errors.empty())
-        return Error(errors);
+        return {errors};
```

Comment:
Please forgo stylistic changes unless you're making functional changes to that code

Reply:
That was also a pedantic warning that Vinnie also always told me apply, but OK.

https://clang.llvm.org/extra/clang-tidy/checks/modernize/return-braced-init-list.html